prober/http.go

```diff
@@ -243,6 +330,38 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 	if resp == nil {
 		resp = &http.Response{}
 	}
+	var (
+		earlierstCertExpiry time.Time
+		i                   = 0
+	)
+	for _, trace := range tt.traces {
+		// We get the duration for the first request from chooseProtocol
+		if i != 0 {
+			durationGaugeVec.WithLabelValues("resolve").Add(trace.dnsDone.Sub(trace.start).Seconds())
+		}
+		if resp.TLS != nil {
+			exp := getEarliestCertExpiry(resp.TLS)
```

Comment:
This is conflicting with the logic below, as there's two different sets of TLS expiration logic. Keep just the existing one that looks only at the last request, for simplicity

Reply:
Oh right, missed that when rebasing.